ADOPTERS.md

```diff
@@ -4,6 +4,9 @@ A list of adopters of Talos Linux, and the use
 
 ## Adopters (listed alphabetically)
 
+* **[Nedap Security Atlas](https://nedapsecurityatlas.com)**  
+  Nedap Security Atlas does all on-premise development, unit- and integrationtesting in Gitlab running on Talos clusters.
```

Comment:
Thanks for the :heart: for Talos. Would you mind us fixing `unit-` to `unit` while signing this commit?

Reply:
no, worries, we'll fix up the signing errors (SideroLabs require all commits to be signed by at-least of the developers, so we'll fix that)